boards/arm/conexio_stratus/board.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 Conexio Technologies, Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
+#include <devicetree.h>
+
+#define GPIO0 DT_LABEL(DT_NODELABEL(gpio0))
+#define SENSOR_POWER_PIN 11 /* ENABLE_3V3_SENSOR --> i2c sensors  */
+
+static int board_conexio_stratus_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	/* Get the device binding */
+	const struct device *gpio = device_get_binding(GPIO0);
+	if (gpio == NULL) {
+		return -ENODEV;
+	}
+
+	/* Configure sensor power pin as output. */
+	gpio_pin_configure(gpio, SENSOR_POWER_PIN, GPIO_OUTPUT_HIGH);
+
+	return 0;
+}
```

Comment:
you can use the regulator API for that (setup a regulator in DT that is always on)

Reply:
Can you help me with this please, possibly with the code snapshot as to how I can set this up in the DT? Thanks

Reply:
@carlescufi, can you assist me as to how can I set up the regulator API in the device tree so that it is always ON for the GPIO PIN number 11. Currently, board_conexio_stratus_init() in board.c was initiating this pin to high to enable the power to the sensors.
Thanks

Reply:
@carlescufi , also the main idea behind this was so that users can dynamically turn on/off the power to the onboard sensors from their application to reduce any idle power consumption when sensors are not in use. Any better way to handle this in the DT?